approved-books.json

```diff
@@ -1,5 +1,13 @@
 [
       {
+    "name": "College Algebra with Corequisite Support",
+    "collection_id": "col32026",
+    "style": "precalculus-coreq",
+    "version": "1.9.12",
+    "server": "cnx.org",
+    "uuid": "507feb1e-cfff-4b54-bc07-d52636cecfe"
```

Comment:
@marcjeannott - It looks like the CI is failing beause the UUID is missing the "3" at the end (e.g. it should be `507feb1e-cfff-4b54-bc07-d52636cecfe3`)

Reply:
Shoot guess my copy paste didn't get it all. I've tried to add the 3 at the end - let me know if it worked (it's my first time doing this).

Reply:
Yep, looks good to me!